packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -101,7 +103,7 @@ const PipelineRuns = ({
             {!pipelineName && (
               <StructuredListCell>
                 <Link
-                  to={urls.pipelineRuns.byPipeline({
+                  to={createPipelineUrl({
```

Comment:
name is a bit misleading, it's creating a URL to PipelineRuns by Pipeline name rather than a URL to a Pipeline.

`createPipelineRunsByPipelineURL`? not sure I like it, but something along those lines